samples/drivers/ublox_sara_n310/src/main.c

```diff
@@ -36,46 +41,48 @@ void main(void)
 	LOG_INF("IP: %s", log_strdup(n310_get_ip()));
 	LOG_INF("Network state: %d", n310_get_state());
 
+	n310_psm_set_csclk(2);
+	n310_psm_config(PSM_ENABLE, PERIODIC_TAU, ACTIVE_TIME);
+	n310_psm_set_mode(PM2);
+
 	memset(&addr, 0, sizeof(addr));
 	addr.sin_family = AF_INET;
 	addr.sin_port = htons(4242);
 	inet_pton(AF_INET, "192.168.0.20", &addr.sin_addr);
 
+	sockfd = socket(AF_INET, SOCK_DGRAM, IPPROTO_UDP);
+	if (sockfd < 0) {
+		LOG_INF("Could not create socket. Return value: %d", sockfd);
+	}
+
+	LOG_INF("Connecting...");
+	n = connect(sockfd, (const struct sockaddr *)&addr, sizeof(addr));
+	if (n < 0) {
+		LOG_ERR("Bind failed with return value: %d", n);
+	}
+
 	while (1) {
 		char *buffer = "ABDD";
 		char recvBuffer[MAX_BUF] = "\0";
-		int sockfd = 0, n = 0;
-
-		sockfd = socket(AF_INET, SOCK_DGRAM, IPPROTO_UDP);
-		if (sockfd < 0) {
-			LOG_ERR("Could not create socket. Return value: %d", sockfd);
-		}
-
-		n = connect(sockfd, (const struct sockaddr *)&addr, sizeof(addr));
-		if (n < 0) {
-			LOG_ERR("Bind failed with return value: %d", n);
-		}
 
+		LOG_INF("Sending...");
 		n = sendto(sockfd, buffer, strlen(buffer) / 2,
 			   RELEASE_AFTER_FIRST_DOWNLINK,
 			   (const struct sockaddr *)&addr, sizeof(addr));
 		if (n < 0) {
 			LOG_ERR("Send failed with return value: %d", n);
 		}
 
-		k_msleep(2000);
+		k_msleep(5000);
```

Comment:
is there a specific reason for these sleep lengths?

Reply:
No profound reason, I was just checking different delays I think. I also worried a little if I was using too much bandwidth while testing.